dll/win32/dbghelp/compat.c

```diff
@@ -174,7 +174,7 @@ BOOL __ReadFile(HANDLE file, PVOID buffer, DWORD len, PDWORD outlen, PVOID overl
 
     read = fread(buffer, 1, len, file);
 
-    if (ferror(file) != 0)
+    if (ferror((FILE*)file) != 0)
```

Comment:
Why would this cast be needed only there and not in all the other f*** function calls where `file` HANDLE is passed while the parameter type is actually a `FILE*` ?

Reply:
This commit is for dbghelp only

Reply:
Yes of course it is for dbghelp, which is precisely this module. Just look at the rest of this file and you'll understand what I mean... (For example, the fread function call just above also takes this `file` parameter, which is in principle a `FILE*`, etc etc.)